syntax = "proto3";
package ni.measurementlink.measurement.tests.nidigital;

message Configurations {
    repeated string pin_names = 1;
    bool multi_session = 2;
}

message Outputs {
    repeated string session_names = 1;
    repeated string resource_names = 2;
    repeated string channel_lists = 3;
    repeated string connected_channels = 4;
    repeated int32 passing_sites = 5;
    repeated int32 failing_sites = 6;
}